src/Arcus.Template.WebApi/Arcus.Template.WebApi.csproj

```
@@ -6,6 +6,10 @@
     <Company>Arcus</Company>
     <RepositoryType>Git</RepositoryType>
     <Description>Provide a template to easily build Web APIs running in Azure.</Description>
+    <IsPackable>true</IsPackable>
+    <PackageId>Arcus.Template.WebApi</PackageId>
```

`Template` or `Templates`?
```suggestion
    <PackageId>Arcus.Templates.WebApi</PackageId>
```

---

Ah, the project calls `Arcus.Template`. Since I find it more logical if you select a project:
`Arcus.Template.WebApi`.

---

Sounds good to me!